src/components/label/label.js

```diff
@@ -0,0 +1,12 @@
+import classNames from './label.module.scss';
```

Comment:
It's not just the label, right? It also has a value

Reply:
Ref : https://github.com/Real-Dev-Squad/website-roadmap/pull/8/commits/08fcb4a54dff1dd083ba1c2e4cc8df90ef496daf